src/main/java/pages/CategoryPage.java

```diff
@@ -0,0 +1,18 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import lombok.Getter;
+
+@Getter
+public class CategoryPage extends PageObject {
+
+    @FindBy(xpath = "//h2[@class='title']//a[contains(text(),'Cell phones')]")
+    private WebElement subcategory;
```

Comment:
use code convention
rename subcategory in subCategory, please

Reply:
Done